src/Input/Silk.NET.Input.Common/Interfaces/ICursor.cs

```diff
@@ -44,5 +70,12 @@ public interface ICursor
         /// <param name="mode">The mode to check for.</param>
         /// <returns>Whether or not this mode is supported.</returns>
         bool IsCursorModeSupported(CursorMode mode);

```

Comment:
How about we just rename this and `IsStandardCursorSupported` to just `IsSupported` as it feels a lot cleaner.